metadataserver/identity/verify.go

```diff
@@ -0,0 +1,126 @@
+package identity
```

Comment:
I'm a little bit confused. Is this code only for testing? Or is it used elsewhere?

Reply:
It was used. It is no longer used. I was on the fence on whether I should remove it or not because I can see having signature verification being potentially handy at some point. I don't have a strong feeling either way. Your call.

Reply:
They seem useful. Maybe just add a comment indicating they're helper functions or something? I can imagine the metatron client consuming these at some point.